packages/miniflare/src/workers/core/entry.worker.ts

```diff
@@ -351,6 +355,156 @@ async function handleScheduled(
 	});
 }
 
+function createForwardableEmailMessage(
```

Comment:
can we please extract these functions into a separate file? They are quite extensive and add to the cognitive load of this file 
🙏